mate-extra/mate-tweak/mate-tweak-22.10.0.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_EXT=1
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+inherit distutils-r1 linux-info
+
+DESCRIPTION="MATE Tweak is a small utility to fine-tune the MATE desktop environment"
+HOMEPAGE="https://github.com/ubuntu-mate/mate-tweak"
+SRC_URI="https://github.com/ubuntu-mate/mate-tweak/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-python/distro
+	dev-python/psutil
+	dev-python/setproctitle
```

Comment:
I think you want to add here `[${PYTHON_USEDEP}]`

Reply:
Thank you! Yes!